src/pages/Login.jsx

```diff
@@ -0,0 +1,53 @@
+import React,{useState,useEffect} from 'react'
+import {Link,useNavigate} from 'react-router-dom'
+import { useAuth } from '../contexts/AuthContext';
+
+export default function Login() {
+  const navigate = useNavigate()
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+  const {LoginHandler } = useAuth();
+ 
+
+  useEffect(() => {
+    (async () => {
+      LoginHandler(loginForm.email, loginForm.password);
+    })();
+  }, [loginForm.email, loginForm.password]);
```

Comment:
this effect will run every time you type in email and password field and hence it runs loginhandler multiple times. This can lead to unexpected issues. So you can call loginhandler on form submit instead of running it with useEffect

Reply:
Didn't think of that. Thank you for pointing it will surely change it.